CONTRIBUTING.md

```diff
@@ -51,14 +57,38 @@ Conditions for whether to Approve or Request Changes for a PR can come with a lo
 Examples of when to Request Changes
 - Logging functions are left in that aren't required or needed later (with exceptions for error logging, i.e. console.error, etc.)
 - Commented-code is left in that isn't needed later
+- Unsafe function calls like `eval()`
 - Widespread incorrect syntax use (using snake case for many variables in JS when it's not required)
 - Significant amount of repeition in code that can be condensed without much issue (code isn't DRY) *
 - Unnecessary function calls, overly verbose code *
 
+## Style Guide
+
+- Avoid using CSS directly. Instead, style through Material UI theming or with CSS-in-JS
+
 ## Common Components/Utilities
 
+### MET API
+
 - `@transational`: Database method decorator. If there is an exception during execution, the entire DB session will be safely rolled back to a point in time just before the decorated function was called. If not, the session will be saved, unless autocommit is set to False. This helps replace most session management boilerplate.
 - `authorization.check_auth`: Checks a user for one or more roles and for the correct engagement.
 - `schema_utils.validate`: The most commonly used method for validating request data in resources. This method takes the request JSON data and compares it to a JSON file that acts as a schema for incoming requests of that type.
 
+### MET Web
+
+- Be sure to make use of shared components found under `src/components/common`. Below is a non-exhaustive list of common app components:
+   - `PrimaryButton`, `SecondaryButton`, `TertiaryButton`: Buttons that come with built-in MET theming and functionality
```

Comment:
The preferred way to use these is to import the one \<Button> component from components/common and use `variant=<"primary"|"secondary"|"tertiary"|undefined>`

Reply:
Thanks for sharing that knowledge! I'll update